frontend/public/components/machine-set.tsx

```diff
@@ -69,9 +69,9 @@ export const getReadyReplicas = (machineSet: MachineSetKind | MachineDeploymentK
 export const getAvailableReplicas = (machineSet: MachineSetKind | MachineDeploymentKind) => _.get(machineSet, 'status.availableReplicas', 0);
 
 const tableColumnClasses = [
-  classNames('pf-m-4-col-on-md', 'pf-m-6-col-on-sm'),
-  classNames('pf-m-4-col-on-md', 'pf-m-6-col-on-sm'),
-  classNames('pf-m-4-col-on-md', 'pf-m-hidden', 'pf-m-visible-on-md'),
+  classNames('col-sm-4', 'col-xs-6'),
```

Comment:
I'm inclined to wonder if there are several exact matches in these `tableColumnClasses` sets that could be shared. For example, this looks the same as the one in `machine-deployment`, `limit-range`, `default-resource`, `cluster-service-plan`...

Reply:
This is obviously a good step, so I wouldn't hold the PR on it.